### ISSUE TITLE
Small improvements to current onboarding flow for new collectives

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#2330]
opencollective.com/create
![](https://d.pr/free/i/7vyMSt+)

Quick short term fixes:
- logo should be fixed (broken right now)
- Default should be "Join Free", not "sign in"

--- 
![](https://d.pr/free/i/2K5a6p+)

- ideally, we shouldn't let the user create an organization account at this point
- FAQ should be contextual to create a collective, not to create an account (make an anonymous donation should go, we should add a question 

---

![](https://d.pr/free/i/kTCbpC+)
- this screen should go. Many people wonder what to choose here and it doesn't matter anyway.

---

![](https://d.pr/free/i/N2JDzQ+)

- Let's get rid of tags here. These can be added later.
- We should add a field for the slug so that people know what is going to be the URL of their collective which will be the first thing they will have to share.

---

![](https://d.pr/free/i/7Bf2vI+)

- We shouldn't go straight to that screen. We should go to the collective page.

---

![](https://d.pr/free/i/rQqnDj+)

- the empty collective page should have clear next steps such as "record a donation", "submit an expense" (but let's keep this for another issue since the collective page is being revamped right now). This also mean that the admin of any collective should be able to record a donation (aka "add funds").

**Comment by znarf** [comment]
@xdamman Can you use the new collective page as basis/screenshot?

**Comment by xdamman** [comment]
The goal of this issue is to show the current flow as of today (and small improvements we can do to it). That's why I wrote for the last screenshot "let's keep this for another issue since the collective page is being revamped right now".

But for the record, here is the current look of the collective page right after creating a new collective:
![](https://d.pr/free/i/BX6gnw+)

**Comment by znarf** [comment]
Ok. In case you want any of these points to be worked on, let's make sure to create separate issues.

**Comment by alanna** [comment]
I agree with all these changes.

If they are creating an account for the first time here as well as creating a Collective, we should also have a platform newsletter opt-in check box.

This might be for a seperate issue but I'll comment here for now:

These changes make it clear how we need to work on the onboarding flow for self-hosting or applying to host once the Collective is created. I really like leaving that question until after the creation process is done, as suggested here.

Once they land on their Collective screen at the end, There can be action buttons in different places of the screen, like 'submit expense' 'create Update' 'create Event' 'edit description' 'update images' etc. 

And also financial related stuff to get them into the host onboarding flow, such as "start accepting financial contributions" and if they click it send them into a flow for choosing or creating their Host. We could have greyed out versions of the financial contribution sections of the new Collective page so they can see what they are activating.
